EventGenerator/src/RPCGun_module.cc

```diff
@@ -169,20 +169,18 @@ namespace mu2e {
     auto output{std::make_unique<StageParticleCollection>()};
     const auto simh = event.getValidHandle<SimParticleCollection>(simsToken_);
     const auto pis = stoppedPiMinusList(simh);
-
+    std::cout<<"event: "<<event.id().event()<<std::endl;
```

Comment:
Please protect this printout with a verbosity flag.

Reply:
yes, probably this needs removing.